self_supervised_pretraining/README.md

```diff
@@ -132,3 +135,24 @@ L1.
 | 6      | 6 | 63.07 | 70.09 | ~11.13% |
 | 12      | 6 | 76.06 | 79.55 | ~4.58% |
 | 24      | 6 | 78.91 | 82.30 | ~4.29% |
+
+### 6. Multi-GPU SSL Training
+
+At the time of creation of this tutorial, the below additional dependencies are required:
+`pip install timm==0.4.12`
+
+To begin training with 2 GPU's please see the below example command for execution of the SSL multi-gpu training
+script:
+
+`CUDA_VISIBLE_DEVICES=0,1 python -m torch.distributed.launch --nproc_per_node=2 mgpu_ssl_train.py --batch_size=8 --epochs=500 --base_lr=2e-4 --logdir_path=/to/be/defined --output=/to/be/defined --data_root=/to/be/defined --json_path=/to/be/defined`
+
+It can be configured to launch on more GPU's by adding the relevant `CUDA Device` ID in `CUDA_VISIBLE_DEVICES`
+and increasing the total count of GPU's `--nproc_per_node`
+
+Below are expected run time comparison between a single GPU and multi-GPU benchmarked on A100 GPU's
+
+
+| Hardware   | Training Time  |
+|------------|----------------|
+| 1 GPU A100 | Under Progress |
```

Comment:
Will the training time for single A100 be updated in the PR

Reply:
Yes

Reply:
Cannot be addressed, due to resource issues as of now.